app/views/password_reset/enter_email.html.haml

```diff
@@ -0,0 +1,14 @@
+%section.forgot_password.container
+  .row
+    .col-sm-10.col-bg-offset-1
+      %form.sign_in{action: '/confirm_password_reset', method: 'post'}
+        %header
+          %h1 Forgot Password?
+          %p We will send you an email with a link that you can use to reset your password.
+        .form-group
+          %label Email Address
+          .row
+            .col-sm-4
+              %input.form-control{name: 'email'}(type="email")
```

Comment:
Does this actually work? I would recommend using either the parentheses or the hash, not both though.